### PR TITLE
Translation sync to add Polish

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**

### CODE REVIEW

File: packages/pyright-internal/src/localization/localize.ts [hunk 1]
```diff
@@ -158,6 +158,8 @@ async function loadStringsFromJsonFile(locale: string): Promise<StringLookupMap
             return (await import(/* webpackChunkName: "ja" */ './simplified.nls.ja.json')).default;
         case 'ko':
             return (await import(/* webpackChunkName: "ko" */ './simplified.nls.ko.json')).default;
+        case 'pl':
+            return (await import(/* webpackChunkName: "nl" */ './simplified.nls.pl.json')).default;
```
Comment (microbit-robert) [inline packages/pyright-internal/src/localization/localize.ts]:
```suggestion
            return (await import(/* webpackChunkName: "pl" */ './simplified.nls.pl.json')).default;
```

**Review by microbit-robert — CHANGES_REQUESTED** [review]
One minor change to fix the webpack chunk name. Otherwise, this looks good.

File: packages/pyright-internal/src/localization/localize.ts (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply packages/pyright-internal/src/localization/localize.ts]: Well that would have been unfortunate! I was completely convinced by seeing the language change from English...

Comment (microbit-matt-hillsdon) [reply packages/pyright-internal/src/localization/localize.ts]: Oh, I see it likely makes no difference...

Comment (microbit-matt-hillsdon) [reply packages/pyright-internal/src/localization/localize.ts]: https://github.com/microbit-foundation/pyright/pull/36/commits/1fc5f9b30c024fa08f2dbfa2effe59b3eb47e141
and updated in the editor PR too

Comment (microbit-robert) [reply packages/pyright-internal/src/localization/localize.ts]: Looks like they just get bundled together?

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**